config.go

```diff
@@ -2,17 +2,20 @@ package main
 
 import (
 	"encoding/json"
+	"fmt"
 	"io/ioutil"
 	"os"
+	"path"
 )
 
 // JiraConfig is a config to communicate with JIRA account.
 type JiraConfig struct {
-	Token   string `json:"token,omitempty"`
-	Email   string `json:"email,omitempty"`
-	UserID  string `json:"user_id,omitempty"`
-	Test    bool   `json:"test,omitempty"`
-	Project string `json:"project"`
+	Token     string `json:"token,omitempty"`
+	Email     string `json:"email,omitempty"`
+	UserID    string `json:"user_id,omitempty"`
+	Test      bool   `json:"test,omitempty"`
+	Project   string `json:"project"`
+	Component string `json:"component"`
```

Comment:
@yaput need to ensure we have default value set as "BACKEND" ya. So, if the config doesn't get filled, other users are not regressed.